user_sync/rules.py

```diff
@@ -269,12 +281,12 @@ def process_dashboard_users(self, dashboard_connectors):
                 for user_key, desired_groups in accessor_unprocessed_groups_by_user_key.iteritems():
                     self.try_and_update_dashboard_user(accessor_organization_info, user_key, dashboard_connector, groups_to_add=desired_groups)
                     
-    def iter_orphaned_dashboard_users(self, orphan_account_types):
+    def iter_orphaned_dashboard_users(self, orphan_account_types=None):
```

Comment:
Don't add a default, we don't need it.  and the one that you added makes this function a no-op, so there's no point in calling it.